components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -701,8 +703,11 @@ impl Interpolate for LengthOrNone {
                 match (from, to) {
                     (&TransformOperation::Matrix(from),
                      &TransformOperation::Matrix(_to)) => {
-                        // TODO(gw): Implement matrix decomposition and interpolation
-                        result.push(TransformOperation::Matrix(from));
+                        let decomposed_from = MatrixDecomposed2D::from(from);
```

Comment:
leave comments that it doesn't yet handle cases where one of the two matrices is not 2D